uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/PopupViewController.h

```diff
@@ -37,8 +42,10 @@ namespace Toolkit
 class PopupViewController : public QObject
 {
   Q_OBJECT
+  Q_PROPERTY(Popup* popup READ popup WRITE setPopup NOTIFY popupChanged)
+  Q_PROPERTY(QAbstractListModel* popupElements READ popupElements NOTIFY popupChanged)
   Q_PROPERTY(PopupManager* popupManager READ popupManager WRITE setPopupManager NOTIFY popupManagerChanged)
-  Q_PROPERTY(QString title READ title NOTIFY popupManagerChanged)
+  Q_PROPERTY(QString title READ title NOTIFY titleChanged)
```

Comment:
I think this was just done for connivence since it was directly linked to the popupManager. The only place `emit popupManagerChanged()` is emitted is in `setPopupManager` which I added `emit titleChanged()` there as well so im pretty confident it will behave the same.